src/seedu/addressbook/data/person/Address.java

```diff
@@ -30,6 +30,14 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
     }
 
     /**
+     * Returns the contact's address in the form as shown below.
+     * Address: 444, delta street
+     */
+    public String getPrintableString() {
```

Comment:
As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.